Update fitness_centre.json

mikini opened the pull request:
* rename "Fitness World" in DK to "PureGym Danmark"
* add PL to "PureGym" of GB (maybe separate entity?)

LaoshuBaby commented:
> * add PL to "PureGym" of GB (maybe separate entity?)

Yes, if it was operated by Poland subsidiary and have a unique wikidata item, a separated entry maybe better.

mikini commented:
Polish locations/subsidiary were closed down [shortly after acquisition in 2020, it appears](https://www.spabusiness.com/wellness-news/PureGym-confirms-closure-of-operations-in-Poland/346627).

Locations currently operated by PureGym are [UK, DK & CH](https://corporate.puregym.com/about-us/overview/default.aspx). KSA & UAE locations are operated under franchise.
A few locations in US are named "Pure Fitness", which is clashing with an [existing HK chain](https://www.wikidata.org/wiki/Q10852204), so haven't done anything about that. 

The Swiss locations seems to [originate from something called Basefit](https://implementconsultinggroup.com/case/basefitch/), acquired by and operated as Fitness World for some years but spun-off into its own PureGym Swiss company now.

I have added PureGym Swiss to wikidata and updated PR to reflect this as three separate entities operating in different geographies, with appropriate matchNames for old names.
